src/front/glsl_new/lex.rs

```diff
@@ -113,6 +113,23 @@ pub fn consume_token(mut input: &str) -> (Option<Token>, &str) {
                 "layout" => (Some(Token::Layout(meta)), rest),
                 "in" => (Some(Token::In(meta)), rest),
                 "out" => (Some(Token::Out(meta)), rest),
+                "flat" => (
+                    Some(Token::Interpolation((meta, crate::Interpolation::Flat))),
+                    rest,
+                ),
+                "noperspective" => (
+                    Some(Token::Interpolation((meta, crate::Interpolation::Linear))),
+                    rest,
+                ),
+                "smooth" => (None, rest),
```

Comment:
why is this `None`?

Reply:
Oops, forgot to change this after adding `Interpolation::Perspective`. Thanks for catching it!